extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1282,9 +1282,7 @@ export class AmpA4A extends AMP.BaseElement {
           'fallback to 3p disabled');
       return Promise.resolve(false);
     }
-    this.promiseErrorHandler_(
-        new Error('fallback to 3p'),
-        /* ignoreStack */ true);
+    dev().warn(TAG, 'fallback to 3p');
```

Comment:
I feel like this is such a common case that it's almost always just going to be noise. Can we maybe downgrade it to `fine` so that it doesn't show up in the dev tools by default?

Reply:
I would want to AMP core to signoff on that first.  It may be a useful signal for them.  For now I want to make sure it does not confuse publishers as a false indicator for incorrectly configuring the ad tag.

Reply:
By "useful signal", do you mean you want it to phone home to the logging server? I guess we can do `warn` for now as a kind of hacky way of doing that, but it would be better to send that using a separate mechanism so that we're not polluting the logs. Maybe add a TODO for this?